server/src/main/java/com/netflix/controller/MovieController.java

```diff
@@ -0,0 +1,77 @@
+package com.netflix.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.orm.jpa.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.netflix.repository.Movie;
+import com.netflix.services.MovieService;
+
+@SpringBootApplication
+@ComponentScan({"com.netflix.services","com.netflix.repository"})
+@RestController
+@RequestMapping(path = "movies")
+@EntityScan("com.netflix.repository")
+@EnableJpaRepositories("com.netflix.repository")
+@EnableAutoConfiguration
+public class MovieController {
```

Comment:
I like that you tried Spring Boot.
But you don't have to put `Configuration` and other annotations on each controller.